mlflow/entities/experiment.py

```diff
@@ -67,6 +97,10 @@ def to_proto(self):
         experiment.name = self.name
         experiment.artifact_location = self.artifact_location
         experiment.lifecycle_stage = self.lifecycle_stage
+        if self.creation_time:
+            experiment.creation_time = self.creation_time
+        if self.last_update_time:
+            experiment.last_update_time = self.last_update_time
```

Comment:
Do we need `if`?

```suggestion
        experiment.creation_time = self.creation_time
        experiment.last_update_time = self.last_update_time
```

Reply:
@harupy here is the error without if checks. To be backward compatible, we need these checks

```
Traceback (most recent call last):
  File "/Users/subras/github/mlflow-trial/local/lib/python3.7/site-packages/flask/app.py", line 2525, in wsgi_app
    response = self.full_dispatch_request()
  File "/Users/subras/github/mlflow-trial/local/lib/python3.7/site-packages/flask/app.py", line 1822, in full_dispatch_request
    rv = self.handle_user_exception(e)
  File "/Users/subras/github/mlflow-trial/local/lib/python3.7/site-packages/flask/app.py", line 1820, in full_dispatch_request
    rv = self.dispatch_request()
  File "/Users/subras/github/mlflow-trial/local/lib/python3.7/site-packages/flask/app.py", line 1796, in dispatch_request
    return self.ensure_sync(self.view_functions[rule.endpoint])(**view_args)
  File "/Users/subras/github/mlflow-trial/local/lib/python3.7/site-packages/mlflow/server/handlers.py", line 454, in wrapper
    return func(*args, **kwargs)
  File "/Users/subras/github/mlflow-trial/local/lib/python3.7/site-packages/mlflow/server/handlers.py", line 524, in wrapper
    return func(*args, **kwargs)
  File "/Users/subras/github/mlflow-trial/local/lib/python3.7/site-packages/mlflow/server/handlers.py", line 980, in _list_experiments
    response_message.experiments.extend([e.to_proto() for e in experiment_entities])
  File "/Users/subras/github/mlflow-trial/local/lib/python3.7/site-packages/mlflow/server/handlers.py", line 980, in <listcomp>
    response_message.experiments.extend([e.to_proto() for e in experiment_entities])
  File "/Users/subras/github/mlflow-trial/local/lib/python3.7/site-packages/mlflow/entities/experiment.py", line 101, in to_proto
    experiment.creation_time = self.creation_time
TypeError: 'NoneType' object cannot be interpreted as an integer
```

Reply:
Thanks for sharing the traceback. Got it, let's keep the current approach.